x/stake/types/validator.go

```diff
@@ -271,16 +271,16 @@ func NewDescription(moniker, identity, website, details string) Description {
 // UpdateDescription updates the fields of a given description. An error is
 // returned if the resulting description contains an invalid length.
 func (d Description) UpdateDescription(d2 Description) (Description, sdk.Error) {
-	if d.Moniker == doNotModifyDescVal {
+	if d2.Moniker == doNotModifyDescVal {
 		d2.Moniker = d.Moniker
 	}
-	if d.Identity == doNotModifyDescVal {
+	if d2.Identity == doNotModifyDescVal {
```

Comment:
Can we export `doNotModifyDescVal`?

Reply:
agreed no reason for this to be unexported - I can muster up some situations where maybe folks will want access to this

Reply:
addressed